src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -513,8 +518,14 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
               )}
 
               {record?.testResults?.map((testResult, index) => (
-                <Box key={index} mb={2} display='flex' alignItems='baseline'>
-                  <Grid item xs={12}>
+                <Box
+                  key={index}
+                  mb={2}
+                  display={isMobile ? 'block' : 'flex'}
+                  alignItems='baseline'
+                  maxWidth={isMobile ? '90%' : '100%'}
```

Comment:
why 90%? this may not work for all mobile form factors?

Reply:
This is because the box doesn't fill the whole space in mobile, and to avoid adding a new grid, but I could try with a new grid if you think that is better

Reply:
Or use a margin right that is the width of the close icon?

Reply:
This is just my observation - feels like we are forcing the desktop layout to also work for mobile - but they are quite different that maybe having two separate markups might be the better way to do this.
I could be wrong of course, so this is fine for now in my relative opinion :)

Reply:
I have accepted the review, feel free to merge - this does work well according to my testing.

Reply:
I will change the margin. I don't feel we need to split it in two markups. It's just displaying the content flex vs block and some spacings, but the elements are all the same. We will have a lot of repeated code if we separate it

Reply:
> I will change the margin. I don't feel we need to split it in two markups. It's just displaying the content flex vs block and some spacings, but the elements are all the same. We will have a lot of repeated code if we separate it

Now that you mention it.. yes. Margin fix looks good, thanks 🙏